public/app/app.bootstrap.js

```diff
@@ -55,7 +55,7 @@
    * @see https://css-tricks.com/snippets/jquery/get-query-params-object/
    */
   var getQueryParameters = function() {
-    return decodeURIComponent(document.location.search).replace(/(^\?)/,'').split('&').map(function(n) {
+    return decodeURIComponent(document.location.search).replace(/(^\?)/,'').split(/[&,?]/).map(function(n) {
```

Comment:
@johncrossman So after we've stripped out the first question mark, another question mark might be present within the query string itself as a delimiter? Is it intentional to also include commas as delimiters?